internal/core/src/exec/Driver.cpp

```diff
@@ -135,6 +147,17 @@ Driver::Run(std::shared_ptr<Driver> self) {
     }
 }
 
```

Comment:
initialize operators before launching the pipeline, this is needed by agg_operator and the same as Velox